Rocket.Chat/Controllers/Chat/MembersListViewController.swift

```diff
@@ -69,9 +69,21 @@ class MembersListViewController: BaseViewController {
     @IBOutlet weak var membersTableView: UITableView!
     var loaderCell: LoaderTableViewCell!
 
-    var data = MembersListViewData()
+    var data = MembersListViewData() {
+        didSet {
+            UIView.performWithoutAnimation {
+                membersTableView?.reloadData()
+            }
```

Comment:
This is causing real weird behaviour on the list... we should be adding new rows instead of reloading the entire tableView. But not required right now. 👍